src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -476,7 +482,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <!-- Output location for publish target. -->
   <PropertyGroup>
-    <PublishDir>$([MSBuild]::EnsureTrailingSlash($([MSBuild]::ValueOrDefault('$(PublishDir)', '$(OutputPath)app.publish'))))</PublishDir>
+    <PublishDir>$([MSBuild]::EnsureTrailingSlash($([MSBuild]::ValueOrDefault('$(PublishDir)', 'publish'))))</PublishDir>
```

Comment:
Why this change? Seems like it would confuse people. Did you mean to make it $(BuildDir)app.publish?

Reply:
No. This was intentional. Making publish a first-class citizen of the MSBuild process. So, I placed it in the root instead.

I've wanted to ask, why the `app.` prefix? why not just `publish`?

Is it because of the explorer's sorting that you wanted to make publish folder come first?